module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -1031,7 +1025,9 @@ public function renewMyItems($renewDetails)
                 $response = $this->makeRequest(
                     'POST',
                     '/circulation/renew-by-id',
-                    json_encode($requestbody)
+                    json_encode($requestbody),
+                    [],
+                    '/.*/'
```

Comment:
Could we also allow something like `true` that would equal `/.*/` but be a bit cleaner?